src/views/AdminEventLog.vue

```diff
@@ -50,14 +68,31 @@
         },
 
         created() {
-            this.getAdminEvents();
+            this.getAllEvents();
         },
 
         methods: {
-          getAdminEvents: async function () {
+          searchEvents: function () {
+            const params = new URLSearchParams();
+            if (this.searchUserId) {
+              params.append('resourcePath', `users/${this.searchUserId}*`);
+            }
+            [
+              {name: 'dateFrom', value: this.searchDateFrom},
+              {name: 'dateTo', value: this.searchDateTo},
+            ].map(param => {
+              if (param.value) params.append(param.name, param.value);
+            });
```

Comment:
This use of the `map` function avoids a long series of `if` statements, which are easy to read but verbose and arguably error prone. Here's it's arguable showy for just two attributes, but makes more sense in the next file, `EventLog.vue`, because there are more fields to map, so I'd like to leave this way it for consistency.